image_proc/cfg/Resize.cfg

```diff
@@ -8,30 +8,30 @@ ID = 'Resize'
 
 gen = ParameterGenerator()
 
-interpolate_enum = gen.enum([gen.const('NN', int_t, 0, 'Nearest neighbor'),
-                             gen.const('Linear', int_t, 1, 'Linear'),
-                             gen.const('Cubic', int_t, 2, 'Cubic'),
-                             gen.const('Lanczos4', int_t, 4, 'Lanczos4')],
-                            'interpolation type')
-gen.add('interpolation', int_t, level=0,
-        description='Interpolation algorithm between source image pixels',
+interpolate_enum = gen.enum([gen.const("INTER_NEAREST", int_t, 0, "Nearest neighbor"),
+                             gen.const("INTER_LINEAR", int_t, 1, "Linear"),
+                             gen.const("INTER_CUBIC", int_t, 2, "Cubic"),
+                             gen.const("INTER_LANCZOS4", int_t, 4, "Lanczos4")],
+                            "interpolation type")
+gen.add("interpolation", int_t, level=0,
```

Comment:
All of these single to double quote conversions seem unnecessary.

Reply:
This single quotes is not consistent with other nodelet config files. I am trying help keep code consistent.